src/main/java/uk/gov/ida/eidas/trustanchor/CountryTrustAnchor.java

```diff
@@ -22,6 +22,10 @@
 public class CountryTrustAnchor {
 
     public static JWK make(List<X509Certificate> certificates, String keyId) {
+        return make(certificates, keyId, true);
+    }
+
+    public static JWK make(List<X509Certificate> certificates, String keyId, Boolean validateKey) {
```

Comment:
I know we rarely comment in our code. But could this be a good place for one?

Reply:
Yeah, it probably is. Is it not a hard rule that we don't comment?